plugins/3-pipenv.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+test -f Pipfile || return 1
+
+pipenv_bootstrap(){
+  run pipenv install --dev
```

Comment:
If you're planning on creating a virtual environment at `./.venv` for `pip` then I would change this line to `run mkdir -p .venv && pipenv install --dev` so that their outputs are identical.